### PR TITLE
Added new enchants to maxed enchant book

### PR DESCRIPTION
**Pull request by SchIaumeyer**

### CODE REVIEW

File: items/MAXED_ENCHANT_BOOK.json [hunk 1]
```diff
@@ -51,16 +53,16 @@
...
+    "§9Thunderbolt VI§9, §9Thunderlord VII§9,",
+    "§9Titan Killer VII§9, §9Triple-Strike V§9,",
+    "§9True Protection I§9, §9Turbo-Cacti V§9,",
     "§9Turbo-Cane V§9, §9Turbo-Carrot V§9,",
     "§9Turbo-Cocoa V§9, §9Turbo-Melon V§9,",
     "§9Turbo-Mushrooms V§9, §9Turbo-Potato V§9,",
     "§9Turbo-Pumpkin V§9, §9Turbo-Warts V§9,",
     "§9Turbo-Wheat V§9, §9Vampirism VI§9,",
-    "§9Venomous VI§9, §9Vicious V",
+    "§9Venomous VI§9, §9Vicious V§9,",
```
Comment (jani270) [inline items/MAXED_ENCHANT_BOOK.json:65]: Please revert that change.

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/MAXED_ENCHANT_BOOK.json (hunk 1 above)
Comment (SchIaumeyer) [reply items/MAXED_ENCHANT_BOOK.json:65]: ohh i fd up

**Review by jani270 — COMMENTED** [review]
Seems technically fine but could you maybe also add the other 2 enchants from the chili pepper collection? (Tabasco & Habanero Tactics)

**Comment by SchIaumeyer** [comment]
If you could rely on the official wiki for once...

**Review by jani270 — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Pull request merged**